src/parser/parser_driver.cpp

```diff
@@ -1936,7 +2023,18 @@ bool ParserDriver::parse(std::istream& stream, ParserMode parserMode)
 
 	_fileContexts.emplace_back(&stream);
 	_file = YaraFile(currentFileContext()->getTokenStream(), _features);
-	return parseImpl();
+	try {
+		auto output = parseImpl();
+		return output;
+	} catch (const ParserError& e) {
+		if (!incompleteMode())
+			throw e;
+		else
+		{
+			std::cerr << "Warning: '" << e.getErrorMessage() << "'\nThis Warning would be an Error if not parsing in Incomplete mode." << std::endl;
+			return true;
```

Comment:
I would appreciate any reviewer's opinien on if this printout is actually wanted. Thank you in advance.

Reply:
As I said earlier, let the user handle the prints themselves. But I'd argue here that it's pointless to show something like this to an user. Incomplete mode is opt-in so user must know what they are doing.

Reply:
I decided to stop printing these Warnings, thank you for the suggestion.